src/main/java/core/basesyntax/model/RightTriangle.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax.model;
+
+public class RightTriangle extends Color {
+    private double firstLeg;
+    private double secondLeg;
+
+    public RightTriangle(String color, double firstLeg, double secondLeg) {
+        super(color);
+        this.firstLeg = firstLeg;
+        this.secondLeg = secondLeg;
+    }
+
+    @Override
+    public double getArea() {
+        return 0.5 * firstLeg * secondLeg;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Figure: right triangle, area: " + getArea() + " sq. units, firstLeg: "
```

Comment:
Avoid using direct calls to 'System.out.println' in your draw method. Consider using a more flexible logging approach or returning the string to be printed by the caller.